Gemfile

```diff
@@ -17,14 +17,12 @@ gem 'sass', '~> 3.7.3'
 gem 'sass-rails'
 gem 'slimmer', '~> 13.1.0'
 gem 'sprockets-rails', '~> 3.2.0'
-gem 'therubyracer', '~> 0.12.0'
 gem 'uglifier'
 gem 'uk_postcode', '~> 2.1.3'
 gem 'asset_bom_removal-rails', '~> 1.0.0'
 
 group :development do
   gem 'better_errors'
-  gem 'binding_of_caller'
```

Comment:
This is an optional dependency of `better_errors`. So removing this will change the available functionality.

Reply:
Thanks, I've left that in now